crm_stage_multi_company/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 ACS0NE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Crm Stage Multi Company",
+    "summary": """
+        This module adds support for multi company on crm stage.""",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACS0NE SA/NV, Odoo Community Association (OCA)",
+    "website": "https://acsone.eu",
```

Comment:
Could you follow OCA conventions like in : https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py

by adding missing entries and modifying website ?